src/Questions.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Question from './Question';
+
+class Questions extends Component {
+  render() {
+    return (
+      <div>
+        <ul>
+          {this.props.list.map(item => (
+            <Question key={item.id} data={item} />
+          ))}
+        </ul>
+      </div>
+    );
+  }
+}
+
+Questions.propTypes = {
+  list: PropTypes.arrayOf(PropTypes.shape({
```

Comment:
Create a file called, for example, types.js where you will define some common props like data from the Question.jsx because now I see the same duplicated code in propTypes of Questins.jsx, Question.jsx and Answer.jsx.
Again, DRY - don't repeat yourself.